test/test.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('assert');
+const got = require('got');
+const cur = require('./../src/index.js');
+
+describe('return type check', () => {
+    const newInstance = new cur.MyServer(8000);
+
+    before(() => {
+
+        newInstance.startServer();
+    });
```

Comment:
There is a special hook called `after` that should be used to clean up after tests. Let's use it to close the server.